### PR TITLE
ACMS-1727: Field UI improvements.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1727](https://backlog.acquia.com/browse/ACMS-1727)

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/composer.json [hunk 1]
```diff
@@ -77,7 +77,8 @@
         "patches": {
             "drupal/core": {
                 "3059955 - It is possible to overflow the number of items allowed in Media Library": "https://www.drupal.org/files/issues/2019-12-28/3082690-80.patch",
-                "3301692: Passing null to parameter to mb_strtolower() is deprecated with PHP 8.1": "https://git.drupalcode.org/project/drupal/-/merge_requests/2598.patch"
+                "3301692 - Passing null to parameter to mb_strtolower() is deprecated with PHP 8.1": "https://git.drupalcode.org/project/drupal/-/merge_requests/2598.patch",
+                "3356894 - Make field selection less overwhelming by introducing groups": "https://gist.githubusercontent.com/chandan-singh7929/d8e20c0432e622ce67318eac58bd7686/raw/0e128aa6a13946c6a4fc18ff7a621bd856e5f5dc/field-ui-improvements.patch"
```
Comment (rajeshreeputra) [inline modules/acquia_cms_common/composer.json:81]: please use D.O url instead personal gist.

**Review by rajeshreeputra — COMMENTED** [review]
Overall looks good.

**Review by rajeshreeputra — APPROVED** [review]
changes looks good to me.

**Review by deepakmishra2 — APPROVED** [review]
looks good

**Review by rajeshreeputra — APPROVED** [review]
Looks good to me.

**Pull request merged**